app/src/main/res/layout/viewholder_news_item.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/viewholder_news_container"
+    android:background="@color/white">
+
+    <RelativeLayout
```

Comment:
Try to use the aim of ConstraintLayout. Avoid using Relative you can do the same with less levels!